config/default.js

```diff
@@ -25,6 +25,13 @@ module.exports = {
     clientSecret: "<clientSecret>",
   },
 
+  emailSubscriptionCredentials: {
+    user: "<RDS_EMAIL>",
+    pass: "<EMAIL PASS GENERATED AFTER 2FA>",
```

Comment:
- let's not use abbreviations if the full text is not too long.

Reply:
changed.